### PR TITLE
Removing gulp-util 

### PR DESCRIPTION
**Pull request by sebworks**
Removing gulp-util

## Changes

- Modified files to remove `gulp-util` from the build workflow. 

## Testing

1. Run `gulp:lint --travis` and ensure the build works as expected.
2. Run `gulp test:acceptance --sauce` and ensure the build attempts to run on Sauce Labs.

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Internet Explorer 8, 9, 10, and 11
- [ ] Edge
- [ ] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: gulp/tasks/test.js [hunk 1]
```diff
@@ -283,7 +282,8 @@ function spawnProtractor( ) {
     }
   }
 
-  if ( gulpUtil.env.sauce === 'true' ) {
+  const commandLineParams = minimist( process.argv.slice( 2 ) ) || {};
+  if ( commandLineParams.sauce === 'true' ) {
```
Comment (anselmbradford) [inline gulp/tasks/test.js]: This is going to be different behavior, isn't it? Before it was an environment variable; after it's a command-line flag, or am I reading that wrong?

Comment (sebworks) [reply gulp/tasks/test.js]: no, it was always a command line flag. We are going to do the same thing for headless. You still need to have your sauce credentials set though.

Comment (anselmbradford) [reply gulp/tasks/test.js]: Ah okay, thanks!

**Comment by anselmbradford** [comment]
Hmm after running `gulp test:acceptance --sauce` I had an edit to package-lock:

screen shot 2018-03-16 at 11 06 45 am

Maybe give npm install a run before merging.

**Pull request merged**